pkg/auth/cli.go

```diff
@@ -47,6 +48,7 @@ func GetLoginFlags(flags *LoginOptions) *pflag.FlagSet {
 	fs.StringVarP(&flags.Username, "username", "u", "", "Username for registry")
 	fs.BoolVar(&flags.StdinPassword, "password-stdin", false, "Take the password from stdin")
 	fs.BoolVar(&flags.GetLoginSet, "get-login", false, "Return the current login user for the registry")
+	fs.BoolVarP(&flags.Verbose, "verbose", "v", false, "Return detailed info to stdout while performing auth")
```

Comment:
Maybe something like
```suggestion
	fs.BoolVarP(&flags.Verbose, "verbose", "v", false, "Write more detailed information to stdout")
```